src/pages/Success.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 
 class Success extends React.Component {
   
   render() {
     return(
-      <div></div>
+      <div className="container">
+        <div className="success-component">
+          <h2>Your driver has received your feedback.</h2>
+          <h2>Thank you for choosing Tigerpool!</h2>
+          <Link to="/" className="btn btn-dark btn-md px-4 mt-4">
+            Back to Home
+          </Link>
+        </div>
+      </div>
```

Comment:
Maybe we can generalize this page? Finishing up the driver registration, I feel we can benefit from a success page there too. It'd be nice if I could also use this and pass in a custom message/title/etc as props into this component to keep a consistent look and feel.